docs/guides/joint_fabric_guide.md

```diff
@@ -250,4 +250,22 @@ jf-admin-app has been installed:
 A `Subjects` field equal to `18446744065119551489` (`FFFFFFFDFFFF0001` in hex)
 should be found.
 
-## Run Joint Fabric Demo
+## Manually Testing JCM Commissioning
+
+Execute all the initialization steps for Ecosystem A and Ecosystem B above.
+
+On JFC B
```

Comment:
Rest of the README is using jf-control-app and jf-admin-app terminology. I would suggest keeping the same naming convention in the entire README file.